splink/blocking_rules_library.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from .blocking import BlockingRule
+from .input_column import InputColumn
+
+
+class exact_match_rule(BlockingRule):
```

Comment:
Just a naming thing - when trying this out I was wondering if this should be shortened to `exact_match`? 

Happy to keep as is if you think it is important to make the naming explicit to try and distinguish more form comparison libraries etc.

Reply:
I didn't name it `exact_match` due to potential clashes in the namespace.

We can't use function overloading here because there's no way of distinguishing between two instances of `exact_match("first_name")`

Reply:
Ah yea, I thought there would be a reason! Happy with it as is then 😊